### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nagyrobi**
Add self-populating dropdown with heat modes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -155,6 +155,12 @@ In the event service call any variable coming from the MQTT message can be used
               entity_id: climate.thermostat_1
               temperature: "{{ val | int / 10 }}"
       - obj: "p2b4"
+        properties:
+          "options": >
+            {% for mode in state_attr('climate.thermostat_1','hvac_modes') %}
+            {{ mode }}{{ "\n"|e }}{%- if not loop.last %}
```
Comment (dgomes) [inline README.md:161]: see if this works:

{{ mode + "\n" | e }}

Comment (nagyrobi) [reply README.md:161]: it works.
I just noticed that there's a space before all items except the first one:
kép

Comment (dgomes) [reply README.md:161]: you would need to  erase all spaces in the template

Comment (nagyrobi) [reply README.md:161]: Cleared everything, but still.
```
            {%for mode in state_attr('climate.halo_klima','hvac_modes')%}
            {{mode+"\n"|e}}{%-if not loop.last%}{%-endif%}{%-endfor%}
```

Comment (dgomes) [reply README.md:161]: :( no clue....

Comment (nagyrobi) [reply README.md]: if it's all in one line it's OK: `{%for mode in state_attr('climate.thermostat_1','hvac_modes')%}{{mode+"\n"|e}}{%-if not loop.last%}{%-endif%}{%-endfor%}`
Updated the patch.

**Review by dgomes — APPROVED** [review]

**Pull request merged**